lib/rules/no-v-html.js

```diff
@@ -6,7 +6,7 @@
 const utils = require('../utils')
 
 // ------------------------------------------------------------------------------
-// Rule Definitionutilu
```

Comment:
Whaat happened here 🙈 😂

Reply:
no clue but editor was screaming at me on this file xd